packages/api/src/feature_flag/controller.test.ts

```diff
@@ -0,0 +1,79 @@
+import request from "supertest";
+import type { NextFunction, Request } from "express";
+import { db } from "../database";
+import { app } from "../app";
+import { extractUserIsAdminFromAuthToken } from "../middleware";
+import type { FeatureFlagRequest } from "../idpuser/models";
+
+jest.mock("../database");
+jest.mock("../middleware");
+jest.mock("../mixpanel");
+
+describe("GET /feature-flags", () => {
```

Comment:
These tests look great, but it's a good idea to also test the error cases to make sure that we send the right status code and log the error correctly if, say, the database call fails.

Reply:
I'd add two more error cases: One where the database call fails when the caller is an admin, and one where the request is invalid.